hbase-client/pom.xml

```diff
@@ -72,6 +72,10 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Hmm, it's not just server but client too can't live without it.

Reply:
I'll be back here. Its an odd one. We depend on hbase-wal in hbase-client JUST because of FailedLogCloseException... I'll probably undo this dependency in a later version.

Reply:
> We depend on hbase-wal in hbase-client JUST because of FailedLogCloseException... I'll probably undo this dependency in a later version.

Oh, that's better than expected. Good to know.
Thanks